core/src/main/java/org/radargun/stages/test/TestStage.java

```diff
@@ -61,8 +63,8 @@
    @Property(doc = "Delay to let all threads start executing operations. Default is 0.", converter = TimeConverter.class)
    public long rampUp = 0;
 
-   @Property(converter = TimeConverter.class, doc = "Time between consecutive requests of one stressor thread. Default is 0.", deprecatedName = "delayBetweenRequests")
-   protected long thinkTime = 0;
+   @Property(converter = DurationConverter.class, doc = "Time between consecutive requests of one stressor thread. Default is 0.", deprecatedName = "delayBetweenRequests")
```

Comment:
I think yes. Why not ?

Reply:
I guess the disconnect for me is that the default is now a `null` Duration object, instead of a `zero`? Maybe the doc should say `Default is a null Duration which means no think time between requests`?

Reply:
Done